drivers/input/input_gt911.c

```diff
@@ -28,10 +27,16 @@ LOG_MODULE_REGISTER(gt911, CONFIG_INPUT_LOG_LEVEL);
 #define TOUCH_STATUS_MSK (1 << 7U)
 
 /* The GT911's config */
-#define REG_GT911_CONFIG   BSWAP_16(0x8047U)
-#define REG_CONFIG_VERSION REG_GT911_CONFIG
-#define REG_CONFIG_SIZE    (186U)
-#define GT911_PRODUCT_ID   (0x00313139U)
+#define REG_GT911_CONFIG            BSWAP_16(0x8047U)
+#define REG_CONFIG_VERSION          REG_GT911_CONFIG
+#define REG_CONFIG_TOUCH_NUM_OFFSET (0x804C - 0x8047)
+#define REG_CONFIG_SIZE             (186U)
+#define GT911_PRODUCT_ID            (0x00313139U)
```

Comment:
don't really need the parenthesis aronud these two, I'd drop them for coherency

Reply:
Done. :)